Create a simple scroll of posts

motuncoded opened the pull request:
In this feature, I  used Posts Placeholder API which  render a simple scroll of different posts from the top to the end of the posts.


Resolves #3

Commit:
Create a simple scroll of posts

In this feature, I used Posts Placeholder Api which  render a simple scroll of different posts.

Asylcreek commented on pages/index.tsx:
```diff
@@ -1,12 +1,80 @@
-import { inter } from "../styles/fonts";
 
 export default function Home() {
+  type Post = {
+    id: number;
+    title: string;
+    body: string;
+    userId: number;
+  };
+
+  type Error = {
+    message: string;
+  };
```
Move these outside the React function

motuncoded replied:
Ok

Asylcreek reviewed (changes requested):
Please install Prettier

iamr0b0tx commented:
> Please install Prettier

This is the second time I have seen this comment.

I think the first was because you didn't install prettier; this one is because you did not run the prettier formatter before pushing this change.

I advise you to set up a hook to ensure your file gets formatted when you change. An example will be an on-save hook, it formats the file when you save it, some IDEs support this. 

Another option will be a pre-commit hook, I don't like this one because I think formatting changes should be reviewed before committing. There are a few ways around this. You can set prettier in your pre-commit hook such that it raises an error when it sees unformatted code, that way when you try to commit it will fail and tell you it is because it found unformatted code. In that case, you can then run the formatter, review the changes, add it and then try committing again. 

__PLEASE ENSURE YOU READ THIS MESSAGE MULTIPLE TIMES AND UNDERSTAND WHAT I AM SAYING, I WOULD HATE TO HAVE TO REPEAT MYSELF__. Hope it helps

Commit:
Setup the prettier and made file changes

In this change, I set up prettier in the project and move the function types outside the Reactjs function

Asylcreek reviewed (approved):
Please deploy, so I can see live previews
